src/kernel/Cargo.toml

```diff
@@ -22,6 +22,7 @@ nonoverlapping_interval_tree = {version = "0.1.3", default-features = false}
 twizzler-abi = { path = "../lib/twizzler-abi", default-features = false, features = ["kernel"]}
 memoffset = "0.6.5"
 limine = "0.1.8"
+intrusive-collections = "0.9.5"
```

Comment:
This is going to be an odd request but, when we add a library like this one where new folks are going to make the tilting dog head (Huh?) we need to figure out a good way to point directly to the documentation for such a feature.  I found this:
https://docs.rs/intrusive-collections/latest/intrusive_collections/ Is that the right place to look?  "Intrusive" (yeah, not your name, but...) makes any reviewer go..."What?" ;-)

Reply:
Should I include a big block comment where we use the feature, in the code?

Reply:
Yeah, that makes sense. Thanks.